tests/helpers.py

```diff
@@ -1,7 +1,17 @@
 import numpy as np
 from itertools import cycle
+import pytest
 
 from mvg.mvg import MVG
+from mvg.exceptions import MVGAPIError
+
+
+def delete_analysis(session: MVG, request_id: str) -> None:
+    # Delete analysis
+    session.delete_analysis(request_id)
+    with pytest.raises(MVGAPIError) as exc:
+        session.get_analysis_results(request_id)
+    assert f"Request with ID {request_id} does not exist" in str(exc)
```

Comment:
This is used as both as a helper function and an actual test function. In the tests that call this function we can instead just use `session.delete_analysis`. And move the assertion parts of this function to a separate test case

Reply:
Good comment. If I understand, the new test case should run an analysis and assert analysis deletion?

Reply:
exactly. I see you have done just that

Reply:
Moreover, it will be great if the file `test_api_analyses` could have test cases named after the analysis endpoint, and not based on the feature (`callback`, `ModeId`, `KPI`)